openapi/mt_openapi_spec_v1.yaml

```diff
@@ -16028,8 +16033,12 @@ components:
           "$ref": "#/components/schemas/foreign_exchange_rate"
           nullable: true
           description: Associated serialized foreign exchange rate information.
+        vendor_attributes:
+          type: object
+          nullable: true
+          description: 'Translation missing: en.openapi.descriptions.payment_order.schema.vendor_attributes'
```

Comment:
translation still missing

Reply:
@rohanshiva Can you add in the description?

Reply:
Sorry! I didn't realize this was missing description. 

Does this work? https://github.com/Modern-Treasury/platform/pull/23704

Reply:
yah! thanks for adding

Reply:
Np! Will merge the PR right after the code freeze.